build-logic/cleanup/src/main/kotlin/gradlebuild.cleanup.gradle.kts

```diff
@@ -16,17 +16,7 @@
 
 import gradlebuild.basics.repoRoot
 import gradlebuild.cleanup.services.DaemonTracker
-import gradlebuild.cleanup.tasks.KillLeakingJavaProcesses
 
-plugins {
-    base
-}
-
-val trackerService = gradle.sharedServices.registerIfAbsent("daemonTracker", DaemonTracker::class) {
-    parameters.gradleHomeDir.fileValue(gradle.gradleHomeDir)
+gradle.sharedServices.registerIfAbsent("daemonTracker", DaemonTracker::class) {
     parameters.rootProjectDir.set(repoRoot())
 }
-
-tasks.register<KillLeakingJavaProcesses>("killExistingProcessesStartedByGradle") {
```

Comment:
It seems like we need to keep this dummy task, otherwise TeamCity parallel test complains....

https://builds.gradle.org/viewLog.html?buildId=55197470&tab=buildResultsDiv&buildTypeId=Gradle_Master_Check_ConfigCacheSantaTrackerSmokeTestsJava11&branch_Gradle_Master_Check_Stage_PullRequestFeedback=blindpirate%2Fre-implement-killLeakingJavaProcesses